doc/build_apps/install_bin.rst

```diff
@@ -61,7 +61,7 @@ The CCF install notably contains:
 
 - The ``cchost`` binary required to spin up a CCF application
 - The ``cmake`` files required to build CCF applications
-- Azure and ``ansible`` scripts required to :doc:`create_vm` and :doc:`build_setup` (under ``getting_started/``)
+- The ``ansible`` playbooks required :doc:`build_setup` (under ``getting_started/``)
```

Comment:
```suggestion
- The ``ansible`` playbooks required for :doc:`build_setup` (under ``getting_started/``)
```
(since `:doc:build_setup` renders as "CCF Development Setup")